ethcore/src/evm/evm.rs

```diff
@@ -66,6 +67,8 @@ pub enum Error {
 	MutableCallInStaticContext,
 	/// Likely to cause consensus issues.
 	Internal(String),
+	/// Wasm runtime error
+	Wasm(String),
```

Comment:
I would avoid to wrap `parity_wasm::interpreter::Error`, because it cannot be later meaningfully encoded as trace error and put in trace db, that's why it's encoded as string

Also I cannot imagine any code that would rely on handling specific wasm interpreter errors